src/lib/useForm.ts

```diff
@@ -32,18 +32,12 @@ interface UseFormOptions<TSchema extends z.ZodType> {
    * The initial state of the form
    */
   initialState?: MaybeRefOrGetter<NullableKeys<z.infer<TSchema>>>
-  /**
-   * Called when the form is attempted to be submitted, but is invalid.
-   * Only called for client-side validation.
-   */
-  onSubmitError?: () => void
```

Comment:
Onzeker of dit de bedoeling is dus comment ma eentje

Reply:
Ja, dat was de bedoeling maar beter in die andere PR gedaan maar soit